packages/database/schema.prisma

```diff
@@ -210,36 +210,42 @@ enum displayOptions {
 }
 
 model Survey {
-  id                  String                  @id @default(cuid())
-  createdAt           DateTime                @default(now()) @map(name: "created_at")
-  updatedAt           DateTime                @updatedAt @map(name: "updated_at")
-  name                String
-  redirectUrl         String?
-  type                SurveyType              @default(web)
-  environment         Environment             @relation(fields: [environmentId], references: [id], onDelete: Cascade)
-  environmentId       String
-  status              SurveyStatus            @default(draft)
+  id                       String                  @id @default(cuid())
+  createdAt                DateTime                @default(now()) @map(name: "created_at")
+  updatedAt                DateTime                @updatedAt @map(name: "updated_at")
+  name                     String
+  redirectUrl              String?
+  type                     SurveyType              @default(web)
+  environment              Environment             @relation(fields: [environmentId], references: [id], onDelete: Cascade)
+  environmentId            String
+  status                   SurveyStatus            @default(draft)
   /// @zod.custom(imports.ZSurveyQuestions)
   /// @zod.custom(imports.ZSurveyQuestions)
   /// [SurveyQuestions]
-  questions           Json                    @default("[]")
+  questions                Json                    @default("[]")
   /// @zod.custom(imports.ZSurveyThankYouCard)
   /// @zod.custom(imports.ZSurveyThankYouCard)
   /// [SurveyThankYouCard]
-  thankYouCard        Json                    @default("{\"enabled\": false}")
-  responses           Response[]
-  displayOption       displayOptions          @default(displayOnce)
-  recontactDays       Int?
-  triggers            SurveyTrigger[]
-  attributeFilters    SurveyAttributeFilter[]
-  displays            Display[]
-  autoClose           Int?
-  delay               Int                     @default(0)
-  autoComplete        Int?
-  closeOnDate         DateTime?
+  thankYouCard             Json                    @default("{\"enabled\": false}")
+  responses                Response[]
+  displayOption            displayOptions          @default(displayOnce)
+  recontactDays            Int?
+  triggers                 SurveyTrigger[]
+  attributeFilters         SurveyAttributeFilter[]
+  displays                 Display[]
+  autoClose                Int?
+  delay                    Int                     @default(0)
+  autoComplete             Int?
+  closeOnDate              DateTime?
   /// @zod.custom(imports.ZSurveyClosedMessage)
   /// [SurveyClosedMessage]
-  surveyClosedMessage Json?
+  surveyClosedMessage      Json?
+  brandColor               String?
+  highlightBorderColor     String?
+  overwriteBorderHighlight Boolean                 @default(false)
```

Comment:
please remove `overwriteBorderHighlight` and use the value of `highlightBorderColor` instead to check if a survey specific highlightBorderColor is set. We want to keep the database variables to a minimum 😊

Reply:
Yes @mattinannt I've considered that as well, but there's a problem. If the global border is set to ON, there's no option to turn it OFF specifically for the survey. Without a boolean, the only alternative is to overwrite the border color, not to disable it.

Reply:
yeah, you are right, haven't thought about that :-)